app/controllers/general.php

```diff
@@ -446,6 +448,20 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
                 requestTimeout: 30
             );
 
+
+            // Branded banner for previews
+            $transformation = new Transformation(new Preview($executionResponse['body']));
+            if ($type === 'deployment' && $transformation->isValid($executionResponse['headers'])) {
+                $transformation->transform();
+                $executionResponse['body'] = $transformation->getOutput();
+
+                foreach ($executionResponse['headers'] as $key => $value) {
```

Comment:
Why are we doing this instead of just setting the content-length directly?

Reply:
This way I can do `tolowercase` during check, and ensure it works even if response from runtime has uppercase letters - headers as per RFC are case-insensitive